example-studio-all-fashion-mnist/code/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Keras==2.4.3
+tensorflow==2.5.0
```

Comment:
no need to have keras separate from tensorflow here. And I assume the user has already installed DVC on their system and don't add a (possibly conflicting) dvc to the requirements